contrib/multivariate-tada/genunivdata.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+import torch
+
+#Simulates univariate data based on demo mutation rates in original TADA code
+
+def genUnivData(gamma, q, N_family, N_case, N_control, mut_1, mut_2):
```

Comment:
Prefer `snake_case` to `camelCase` for method and variable names: https://pep8.org/#function-names

Reply:
Also, it's fine to use single character variables in order to maintain consistency with notation used in a paper, but if so we'll want to gloss them somewhere, probably in the module docstring.